pascals-triangle.json

```diff
@@ -0,0 +1,47 @@
+{
+    "#": [
+      "Expectations are represented here as an array of arrays.",
+      "How you represent this idiomatically in your language is up to you.",
+      "An expectation of -1 indicates some sort of failure should occur"
+    ],
+    "rows": {
+      "description": "Given a count, return a collection of that many rows of pascal's triangle",
+      "cases": [
+        {
+            "description": "single row",
+            "count": 1,
+            "expected": [[1]]
+        },
+        {
+            "description": "two rows",
+            "count": 2,
+            "expected": [[1], [1, 1]]
+        },
+        {
+            "description": "three rows",
+            "count": 3,
+            "expected": [[1], [1, 1], [1, 2, 1]]
+        },
+        {
+            "description": "negative rows",
+            "count": -1,
+            "expected": -1
+        },
+        {
+            "description": "no rows",
+            "count": null,
+            "expected": -1
+        }
+      ]
+    },
+    "last_row": {
+      "description": "Returns the last row of the pascal's triangle",
```

Comment:
If someone were to asked me to come up with a point, I would say: Someone might argue that having `last_row` would mean that the code to generate the row need only store the current row and the previous row, instead of all rows (as it would have to in `rows`).

Such a consideration may not be a concern for the target audience. I'm not opposed to removing `last_row`.